app/services/katello/pulp/pulp_content_unit.rb

```diff
@@ -0,0 +1,55 @@
+require 'set'
+
+module Katello
+  module Pulp
+    class PulpContentUnit
+      #  Any class that includes this module should define:
```

Comment:
This says they should define whats below, but 2 of the 3 are listed as optional which seems conflicting to me as a developer trying to add a new type.